kubernetes_practice/resources/server.py

```diff
@@ -0,0 +1,41 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+"""
+This is a sample http server
+"""
+
+class Request_handler(BaseHTTPRequestHandler):
+    def _set_headers(self):
+        self.send_response(200)
+        self.send_header('Content-type', 'text/html')
+        self.end_headers()
+
+    def do_GET(self):
+        self._set_headers()
+        self.wfile.write(bytes("<p>hello</p>", "utf-8"))
+
+    def do_HEAD(self):
+        self._set_headers()
+
+
+def run(server_class=HTTPServer, *, handler_class=Request_handler, port=8088):
+    """
+    运行server
+    :param server_class:
+    :param handler_class:
+    :param port:
+    :return:
+    """
+    server_address = ('localhost', port)
+    httpd = server_class(server_address, handler_class)
+    print('Starting httpd...')
```

Comment:
Using `print` here is fine.

Just want to point out that for more serious coding, use the python `logging` module instead of just printing. This is a good learning subject for you.